artifactory/services/fspatterns/utils.go

```diff
@@ -32,9 +32,14 @@ func GetPaths(rootPath string, isRecursive, includeDirs, isSymlink bool) ([]stri
 
 // Transform to regexp and prepare Exclude patterns to be used
 func PrepareExcludePathPattern(params serviceutils.FileGetter) string {
+	exclusions := params.GetExclusions()
+	if len(exclusions) == 0 {
+		exclusions = params.GetExcludePatterns() // Support legacy excluded patterns
```

Comment:
Let's put the description above the line for better readability. Also let's explain that 'excludePatterns' are deprecated and replaced replaced by 'exclusions'